app.js

```diff
@@ -26,14 +34,15 @@ let run = true;
 const streamers = [];
 
 // ========================================== CONFIG SECTION =================================================================
-const CONFIG_PATH = "config.json";
+const CONFIG_PATH = "./config.json";
+let CONFIG_FILE = fs.existsSync(configPath) ? JSON.parse(fs.readFileSync(configPath, 'utf8')) : null;
```

Comment:
I'd rather prefer we kept this in its own async function instead of doing I/O at the top level

Reply:
We can't because I am actually using it everywhere to then print the game name.

Reply:
We can pass in the game names as a parameter, I dislike non-constant globals

Reply:
well ok, but how can I modify the push?